include/net/nrf_cloud.h

```diff
@@ -565,11 +565,9 @@ int nrf_cloud_uninit(void);
  * If it is received before @ref NRF_CLOUD_EVT_TRANSPORT_CONNECTED,
  * the application may repeat the call to @ref nrf_cloud_connect to try again.
  *
- * @param[in] param Parameters to be used for the connection.
- *
  * @retval Connect result defined by enum nrf_cloud_connect_result.
  */
-int nrf_cloud_connect(const struct nrf_cloud_connect_param *param);
+int nrf_cloud_connect(void);
```

Comment:
you might want to save this change for a separate PR since it will require updating all places in NCS that call `nrf_cloud_connect`

Reply:
Or at a minimum, it should be a separate commit. But separate PR makes more sense, yeah.